torchmdnet/datasets/__init__.py

```diff
@@ -1,8 +1,10 @@
+from torchmdnet.datasets.ani1x import ANI1X
```

Comment:
Why import it twice?

Reply:
I think that was added by vscode. I must have missed it.